Suggest/spreadsheet: 21.3 Google Sheets, make gs4_deauth() explicit (#1473)
hadley/r4ds: spreadsheets.qmd

### Authentication

> While you can read from a public Google Sheet with `gs4_deauth()`: without authenticating with your Google account, reading a private sheet or writing to a sheet requires authentication so that googlesheets4 can view and manage *your* Google Sheets.

Reviewer: ```suggestion
While you can read from a public Google Sheet without authenticating with your Google account and with `gs4_deauth()`, reading a private sheet or writing to a sheet requires authentication so that googlesheets4 can view and manage *your* Google Sheets.
```

> When you attempt to get authentication with `gs4_auth()`, googlesheets4 will direct you to a web browser with a prompt to sign in to your Google account and grant permission to operate on your behalf with Google Sheets.

Reviewer: ```suggestion
When you attempt to read in a sheet that requires authentication, googlesheets4 will direct you to a web browser with a prompt to sign in to your Google account and grant permission to operate on your behalf with Google Sheets.
```

> However, if you want to specify a specific Google account, authentication scope, etc. you can do so, e.g., with `gs4_auth(email = "mine@example.com")`, which will force the use of a token associated with a specific email.

Reviewer: ```suggestion
However, if you want to specify a specific Google account, authentication scope, etc. you can do so with `gs4_auth()`, e.g., with `gs4_auth(email = "mine@example.com")`, which will force the use of a token associated with a specific email.
```

> However, if you want to specify a specific Google account, authentication scope, etc. you can do so with `gs4_auth()`, e.g., with `gs4_auth(email = "mine@example.com")`, which will force the use of a token associated with a specific email.

Reviewer: ```suggestion
However, if you want to specify a specific Google account, authentication scope, etc. you can do so with `gs4_auth()`, e.g., `gs4_auth(email = "mine@example.com")`, which will force the use of a token associated with a specific email.
```
